src/cmd/tag.go

```diff
@@ -120,28 +81,25 @@ opslevel get tag --type=Service ID|ALIAS KEY | jq
 		err := validateResourceTypeArg()
 		cobra.CheckErr(err)
 
-		resource := args[0]
+		resourceIdentifier := args[0]
 		tagKey := args[1]
 
-		var result any
-		if opslevel.IsID(resource) {
-			id := opslevel.ID(resource)
-			result, err = TaggableResourceFetchFunctions[opslevel.TaggableResource(resourceType)](id)
-		} else {
-			alias := args[0]
-			result, err = TaggableResourceFetchAliasFunctions[opslevel.TaggableResource(resourceType)](alias)
-		}
-
-		tags, err := GetTags(result)
+		client := getClientGQL()
+		result, err := client.GetTaggableResource(opslevel.TaggableResource(resourceType), resourceIdentifier)
 		cobra.CheckErr(err)
 
-		output := []opslevel.Tag{}
+		tags, err := result.GetTags(client, nil)
+		cobra.CheckErr(err)
+		foundTag := opslevel.Tag{}
 		for _, tag := range tags.Nodes {
-			if tagKey == tag.Key {
-				output = append(output, tag)
+			if tag.Key == tagKey {
+				foundTag = tag
+				break
 			}
 		}
 
+		output := []opslevel.Tag{foundTag}
```

Comment:
If there's no found tag, here's what I get on my machine:

```
~/repos/cli/src db/implement…es-interface ❯ opslevel get tag --type service $SERVICE ab                              Ruby 3.0.0 04:05:47 pm
[
  {
    "id": null,
    "key": "",
    "value": ""
  }
]

```

The output I expected was `[]` (empty JSON array).

Reply:
There can be a case where a key has multiple values

```
~/repos/cli/src db/implement…es-interface ❯ opslevel list tag --type team platform                                   Ruby 3.0.0 04:09:04 pm
[
  {
    "id": "Z2lkOi8vb3BzbGV2ZWwvVGFnLzYwNDI5NzAx",
    "key": "k",
    "value": "v"
  },
  {
    "id": "Z2lkOi8vb3BzbGV2ZWwvVGFnLzYwNDI5NzA2",
    "key": "k",
    "value": "v2"
  },
  {
    "id": "Z2lkOi8vb3BzbGV2ZWwvVGFnLzYwNDI5ODE5",
    "key": "k",
    "value": "v3"
  },
  ...
```

When a user tries to get `k`, the output is:

```
~/repos/cli/src db/implement…es-interface ❯ opslevel get tag --type team platform k                                  Ruby 3.0.0 04:09:14 pm
[
  {
    "id": "Z2lkOi8vb3BzbGV2ZWwvVGFnLzYwNDI5NzAx",
    "key": "k",
    "value": "v"
  }
]

```

But the correct output would be a JSON array of objects, where each object is a tag where `Tag.key` == `keyArg`